tests/integration/fees.test.js

```diff
@@ -0,0 +1,723 @@
+    await withPaidFeeMatchingSnapshot(() => {
+      return dock.accumulatorModule.addUniversalAccumulator(
+        accumulatorIdUni,
+        accumulatedUni,
+        [did, 1],
+        10000,
+        did,
+        pair,
+        1,
+        { didModule: dock.did },
+        false
+      );
+    });
+
+    const start = 10;
+    // The following isn't correct logically but is good enough for getting transaction pricing
+    const accumulated = u8aToHex(accumulatorPos.accumulated);
+    for (let i = 1; i <= 5; i++) {
+      const members = [];
+      for (let j = 0; j < i; j++) {
+        const member = Accumulator.encodePositiveNumberAsAccumulatorMember(
+          start * 10 * i + j
+        );
+        members.push(member);
+      }
+      let witUpd = u8aToHex(
+        WitnessUpdatePublicInfo.new(
+          hexToU8a(accumulated),
+          members,
+          [],
+          kp.secretKey
+        ).value
+      );
+      await withPaidFeeMatchingSnapshot(async () => {
+        await dock.accumulatorModule.updateAccumulator(
+          accumulatorIdPos,
+          accumulated,
+          {
+            additions: members.map((m) => u8aToHex(m)),
+            witnessUpdateInfo: witUpd,
+          },
+          did,
+          pair,
+          1,
+          { didModule: dock.did },
+          false
+        );
+      });
+
+      witUpd = u8aToHex(
+        WitnessUpdatePublicInfo.new(
+          hexToU8a(accumulated),
+          [],
+          members,
+          kp.secretKey
+        ).value
+      );
+
+      await withPaidFeeMatchingSnapshot(async () => {
+        await dock.accumulatorModule.updateAccumulator(
+          accumulatorIdPos,
+          accumulated,
+          {
+            removals: members.map((m) => u8aToHex(m)),
+            witnessUpdateInfo: witUpd,
+          },
+          did,
+          pair,
+          1,
+          { didModule: dock.did },
+          false
+        );
+      });
+    }
+
+    await withPaidFeeMatchingSnapshot(() => {
+      return dock.accumulatorModule.removeAccumulator(
+        accumulatorIdPos,
+        did,
+        pair,
+        1,
+        { didModule: dock.did },
+        false
+      );
+    });
+  }, 30000);
+
+  test("transfers", async () => {
+    const BOB = "5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty";
+
+    await withPaidFeeMatchingSnapshot(() =>
+      dock.signAndSend(dock.api.tx.balances.transfer(BOB, 100), false)
+    );
+
+    const txs = Array(3).fill(dock.api.tx.balances.transfer(BOB, 100));
+
+    const txBatch = dock.api.tx.utility.batch(txs);
+
+    await withPaidFeeMatchingSnapshot(() => dock.signAndSend(txBatch, false));
+  }, 30000);
+
+  test("democracy.notePreimage", async () => {
+    const runtimeVersion = await getRuntimeVersion();
+    const specVersion = runtimeVersion.specVersion.toNumber();
+
+    if (process.env.DISABLE_DEMOCRACY_PREIMAGE_TESTS || specVersion < 44) {
+      return;
+    }
+
+    const preimage = randomAsHex(2e6);
```

Comment:
Although you have explained me the intention of these tests in DM, it will be better to put a comment here about the same.